### PR TITLE
AlenaBurak/4-27-unit-tests-for-questionsservices-2

### PR DESCRIPTION
**Pull request by ElenaBurak**
added question service tests

### CODE REVIEW

**Review by ExadelSandbox — APPROVED** [review]

File: exalearn-api/ExaLearn.Bl/Services/QuestionService.cs [hunk 1]
```diff
@@ -29,8 +29,8 @@ public async Task<TestDTO> GenerateTestAsync(LevelType level)
             var topics = _mapper.Map<TopicQuestionDTO[]>(await _questionRepository.GetTopicsAsync());
 
             return _mapper.Map<TestDTO>(grammarQuestions)
-                .Map(auditionQuestions)
-                .Map(topics);
+                          .Map(auditionQuestions)
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Bl/Services/QuestionService.cs]: what are the spaces?

File: exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+using Xunit;
+
+namespace ExaLearn.Tests.Services
+{
+    public class QuestionServiceTests
+    {
+        private readonly Mock<IQuestionRepository> _mockQuestionRepository;
+        private readonly Mock<IPassedTestRepository> _mockPassedTestRepository;
+        private readonly IMapper _mapper;
+        private readonly QuestionService _questionService;
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs]: IQuestionService

File: exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+                        },
+                         new Answer{
+                        Text="2",
+                        IsCorrect=false,
+                        },
+                          new Answer{
+                        Text="3",
+                        IsCorrect=false,
+                        },
+                           new Answer{
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs]: ctrl+k+d, why is the code not clean?

File: exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+                LevelType = LevelType.Beginner,
+                Question = "AuditionQuestion",
+                Url = "aofvhajfvadfjb",
+                Answers = new List<AnswerDTO>
+                {
+                    new AnswerDTO{
+                    Text="1",
+                    IsCorrect=false,
+                    },
+                     new AnswerDTO{
```
Comment (romanstat) [inline exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs]: styling

Comment (ElenaBurak) [reply exalearn-api/ExaLearn.Tests/Services/QuestionServiceTests.cs]: not working, but i done

**Review by ExadelSandbox — APPROVED** [review]

**Review by romanstat — APPROVED** [review]

**Pull request merged**